templates/main.php

```diff
@@ -17,7 +17,7 @@
 ]);
 ?>
 
-<div id="app" class="announcementcenter" data-is-admin="<?php if (!empty($_['isAdmin'])) { p(1); } else { p(0); } ?>">
+<div id="app" class="announcementcenter" data-is-admin="<?php p(!empty($_['isAdmin']) ? 1 : 0); ?>">
```

Comment:
yeah, but I excluded it in the core PR afterwards. we have to many lines with more then one control-structure.